### ISSUE TITLE
[BUG] <title> Minicpm does not support when aspect ratio gets too high?

### ISSUE THREAD

**Issue by GxTeo** [OpenBMB/MiniCPM-V#706]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

Hosting minicpm-v2.6 with ollama, when my image has an aspect ratio roughly 8, the image itself is fairly wide. I faced the error where the model was unable to generate an image description. Lowering the aspect ratio by padding the images worked or just resizing the image also allow for it to work

### 期望行为 | Expected Behavior

In the README, minicpm-v2.6 is described to be able to process images at any aspect ratio.

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by LDLINGLINGLING** [comment]
hello，This is because the implementation method of Ollama is not exactly the same as that of our official version.
